CHANGELOG.md

```diff
@@ -1,5 +1,27 @@
 # Azure Kubernetes Service Changelog
 
+## Release 2021-10-08
+
+This release is rolling out to all regions - estimated time for completed roll out is 2021-10-14 for public cloud and 2021-10-21 for sovereign clouds.
+
+### Release Notes
+
+* Features
+  * General Availability of Ultra SSD support
+* Preview Features
+  * Public Preview of Private DNS sub zone support for Private Clusters
+  * Public Preview of HTTP Proxy
+  * Public Preview of support for WASM/WASI based nodepools
```

Comment:
worth adding the docs page https://docs.microsoft.com/en-us/azure/aks/use-wasi-node-pools